### DISCUSSION TITLE
Add Enterprise account (Github.com) support

### DISCUSSION THREAD

**Discussion by imbushuo** [community/community#4100] [Enterprise]
Right now the mobile client doesn't work organization that needs Enterprise login (for example, the Microsoft organization under Microsoft Open Source Enterprise), makes it inconvenient to view issues with mobile device (since browser will pull up the app anyway, then the app fails to load data with unknown error.)

**Comment by aconsuegra** [comment]
Hi @imbushuo. I need some more information to fully understand the issue.

Do you mean there is no way to authorize an account via SAML? Or that GitHub Mobile doesn't support enterprise accounts (GHES)?

**Comment by imbushuo** [comment]
Hi @aconsuegra, I think I mean Enterprise on Github.com, not GHES. Specifically, I mean Github.com repos need to authenticate via this screen: https://github.com/microsoft/wslg (I see you are in MS org too so you can probably try it out)

![image](https://user-images.githubusercontent.com/6284353/121082116-79f23080-c792-11eb-909e-bd969b31fd74.png)

Such repo fails to open on mobile client with a single message of "error occurred".

**Reply by aconsuegra** [reply]
Thanks for following up @imbushuo!

You can authorize all your orgs only after a successful login into the app, so you need to logout and log back in (you will see a screen where you can auth to the Microsoft org then). We know this is not a good user experience, and we should allow to auth orgs within the app, we'll work on it soon but I can't give you an ETA.

**Reply by imbushuo** [reply]
Yeah that works. I remembered I joined MS org _after_ logging into mobile client, so that's probably why it doesn't prompt me for login. Logout and login (and then authenticate AAD account) works for me.

**Reply by XVincentX** [reply]
That worked for me too. I am wondering what happens when the token expires and I get kicked out.
